lib/index.js

```diff
@@ -1,7 +1,17 @@
 'use strict';
 
+function _interopDefault (ex) {
+  return (ex && (typeof ex === 'object') && 'default' in ex) ? ex['default'] : ex;
+}
+
 var utils = require('./utils');
 
+var Promise = _interopDefault(require('pouchdb-promise'));
+var pouchdbAjax = _interopDefault(require('pouchdb-ajax'));
+var pouchdbUtils = require('pouchdb-utils');
+
+var inherits = require('inherits');
+
```

Comment:
Can I ask why were are including _interopDefault here? This looks like generated Babel/Rollup code and should not be in source code ideally...

Reply:
This indeed does look like generated code because I copied it from some Rollup generated code.

Currently, the source code uses CommonJS like modules and browserify to be build for browser. However, `pouchdb-ajax` is an ES6 module and uses `default` export. Thus, when `pouchdb-authentication` is used in a Webpack context, it won't be able to dereference the `default` property to the ajax function. This `__interopDefault` handles both the cases where `pouchdb-ajax` is viewed as an ES6 modules (browserify, webpack, ...) and as a CommonJS module (as it is in a Node.js context).

Ideally, `pouchdb-authentication` should be built in two versions as are the plugins in the `pouchdb` monorepo, but that is not the case yet. (Indeed I intend to PR a move to ES6+rollup at some point.)

Reply:
@nolanlawson Do you think there is a better way to do that ? (apart from moving immediately to a ES6+rollup setup ?)

Maybe I could leave a comment that it has to be removed when moving ES6+rollup ?